pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -131,6 +131,8 @@ stdenv.mkDerivation ({
     "-DCOMPILER_RT_ENABLE_IOS=OFF"
   ]) ++ lib.optionals (lib.versionAtLeast version "19" && stdenv.isDarwin && lib.versionOlder stdenv.hostPlatform.darwinMinVersion "10.13") [
     "-DSANITIZER_MIN_OSX_VERSION=10.10"
+  ]  ++ lib.optionals (!haveLibc || bareMetal || isMusl || isDarwinStatic && lib.versionAtLeast release_version "20") [
```

Comment:
Can we pull `!haveLibc || bareMetal || isMusl || isDarwinStatic` into a variable?  Otherwise people are going to forget to change it in both places.

Reply:
Does [this force-push](https://github.com/NixOS/nixpkgs/commit/0cabdde1cea306c99fcc68aa10294ac370443bda) properly address this?